packages/react-sdk/src/components/image-dev.stories.tsx

```diff
@@ -0,0 +1,169 @@
+// Story for image development, see https://github.com/webstudio-is/webstudio-designer/issues/387
+
+import React, { type ComponentProps, type HTMLAttributes } from "react";
```

Comment:
No default import please. Use only named.

Reply:
lets add an eslint plugin for this if it exists?

Reply:
Mb I'm wrong but since React 17 we don't need to import * as React in every place jsx is used?
I haven't written on React for 1-2 years soon, and mb its tooling dependent but even ts shows me that React is unused

Reply:
I would leave it for next PRs as now it's in every place I see in sdk. Easier to fix all in a single PR.

Added here https://github.com/webstudio-is/webstudio-designer/issues/397#issuecomment-1304891614